_posts/2013-04-09-asset-compilation.md

```diff
@@ -17,6 +17,46 @@ templates with `assets/images/logo.png` or in stylesheets with
 
 This functionality of Ember-CLI comes from [broccoli-asset-rev](https://github.com/rickharrison/broccoli-asset-rev). Be sure to check out all the options and usage notes.
 
+### JS Transpiling
+
+Ember-cli automatically transpiles future javascript (ES6/ES2015, ES2016 and beyond) into standard ES5
```

Comment:
- [ ] We should briefly outline that this is using babel 5 (i saw you provided links to the appropriate docs, awesome)
- [ ]  We should point to the babel 6 meta issue, for progress getting to babel 6. For those who will likely be immediately curious